amundsen_application/static/js/components/SearchPage/SearchFilter/index.tsx

```diff
@@ -73,6 +73,7 @@ export const mapStateToProps = (state: GlobalState) => {
         type: categoryConfig.type,
       };
       if (categoryConfig.type === FilterType.CHECKBOX_SELECT) {
+        // eslint-disable-next-line @typescript-eslint/dot-notation
```

Comment:
What was the issue with this line? I've updated this in the filter PR and want to understand if the issue will still be applicable with the change I made.

Reply:
Totally!
Our ESLint configuration includes a preference for the dot notation instead of the brackets, and running ESLint with the --fix flag keeps on chaning this line. 

However, when that change happens, it breaks the types. I am not sure if we are using the bracket notation just for that matter, so I commented it for now. I think that underlining this there is a type safety issue.

Reply:
Got it. I'd have to look into it myself as well to understand what the long term solution is, so disabling the lint for this line makes sense.